citations-tool/tool/src/webapp/vm/citation/_listNestedCitations.vm

```diff
@@ -19,7 +19,7 @@
 		<li id='$linkId' class='months h1Editor' data-value='$nestedSection.getValue()' data-location='$nestedSection.getLocation()'
 			data-sectiontype='$nestedSection.getSectiontype()'>
 			<div id='$editorDivId' class='editor h1Editor'>
-				$nestedSection.getValue()
+				#if($nestedSection.getValue())$nestedSection.getValue()#end
```

Comment:
In velocity you can use `$!nestedSection.value` which will only output the value if it's defined.
http://velocity.apache.org/engine/devel/user-guide.html#quietreferencenotation